frontend/apps/reader/modules/readerfont.lua

```diff
@@ -163,8 +163,8 @@ function ReaderFont:onReadSettings(config)
     self.gamma_index = config:readSetting("gamma_index")
                     or G_reader_settings:readSetting("copt_font_gamma")
                     or DCREREADER_CONFIG_DEFAULT_FONT_GAMMA
-                    or 15 -- gamma = 1.0
-    self.ui.document:setGammaIndex(self.gamma_index)
+                    or 16 -- gamma = 1.0
+    self.ui.document:setGammaIndex(self.gamma_index - 1) -- crengine counts index from 0
```

Comment:
Does this mean that past settings will be shifted by -1 ? That is, users won't get the same gamma value they are used to after this is merged ?
(When I said I was fine with a shift by -1, I was fine about it being done in the code here and there, not about it affecting users. If that would be the case, and there's no other solution, then yes, a one-time migration is needed.)